manifests/init.pp

```diff
@@ -463,9 +463,6 @@
 #                                           can query the puppet-admin-api endpoint
 #                                           Defaults to [ '127.0.0.1', '::1', $::ipaddress ]
 #
-# $server_enable_ruby_profiler::            Should the puppetserver ruby profiler be enabled?
```

Comment:
Can you list this in the changelog under breaking changes or do you want to write those when we release?